suite-common/wallet-core/src/discovery/discoveryThunks.ts

```diff
@@ -239,16 +239,25 @@ const getBundleThunk = createThunk(
 export const getAvailableCardanoDerivationsThunk = createThunk(
     `${DISCOVERY_MODULE_PREFIX}/getAvailableCardanoDerivations`,
     async (
-        { deviceState, device }: { deviceState: string; device: TrezorDevice },
+        {
+            deviceState,
+            device,
+            // isUseEmptyPassphraseForced will be removed once the suite-native has support for passphrase authorization.
```

Comment:
Please, can you add `TODO:` to these comments so we can search for all todos later? I think it is a good way to mark all code that is not meant to be final.

Reply:
I agree with you, but TODO: is overused in this repository.If I filter it, it shows me 500 references :D. 
![Screenshot 2023-11-09 at 12 33 54](https://github.com/trezor/trezor-suite/assets/26143964/ef3b34a5-2fd6-4bc4-875f-4d93c35fbf26)

But if you insists, I can put it there.

Reply:
Yeah, far from perfect, but still better than nothing :) I do not insist though, up to you.

Reply:
Ok, i will put it there . 2c53da92fd